File: snippets/java/src/main/java/org/robolectric/snippets/java/MyActivity.java
package org.robolectric.snippets.java;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class MyActivity extends Activity {
  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

    setContentView(R.layout.activity_my);

    findViewById(R.id.button)
            .setOnClickListener((view) -> ((TextView) findViewById(R.id.text)).setText("Robolectric Rocks!"));
  }
}

File: snippets/kotlin/src/main/java/org/robolectric/snippets/kotlin/MyActivity.kt
package org.robolectric.snippets.kotlin

import android.app.Activity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MyActivity : Activity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    setContentView(R.layout.activity_my)

    findViewById<Button>(R.id.button).setOnClickListener {
      findViewById<TextView>(R.id.text).text = "Robolectric Rocks!"
    }
  }
}
